.goreleaser.yml

```diff
@@ -0,0 +1,33 @@
+builds:
+  -
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+    main: ./cmd/proxy/main.go
+    binary: athens-proxy
+    env:
+      - CGO_ENABLED=0
+archive:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm
+    }}v{{ .Arm }}{{ end }}'
+  format: tar.gz
+  files:
+  - licence*
+  - LICENCE*
+  - license*
```

Comment:
This might be a dumb question, but was the duplication of license intentional? (it is there 4 times, twice uppercase and lowercase)

Reply:
Not dumb at all! Copy & paste error. Thanks!